.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "stage": 1
```

Comment:
Can you double check that this also works in `prod-module-boilerplate`? Make a new issue there if it doesn't?